eclipse-collections/src/main/java/org/eclipse/collections/impl/collector/SummaryStatistics.java

```diff
@@ -25,6 +25,8 @@
 
 /**
  * A Summarizer can be used to aggregate statistics for multiple primitive attributes.
+ *
+ * @since 8.1
  */
 public class SummaryStatistics<T> implements Procedure<T>
```

Comment:
I had thought it would be less than ideal, but considering all the work (including using reflection) that it will take to make the other JDK SummaryStatistics classes Serializable, it may be an ok compromise.